stable/prometheus-operator/hack/sync_prometheus_rules.py

```diff
@@ -105,6 +105,9 @@ def new_representer(dumper, data):
     'alertmanager-$1': {
         'replacement': '$1',
         'init': ''},
+    'https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#': {
```

Comment:
Could you remove the hash from the URL for the replacement?  Please don't forget to update the README.

Reply:
I left the hash in so runbook prefixes could allow for several options:
`https://example.com/runbook#alert` for single page run books
`https://example.com/runbook/alert` for multiple page run books
`https://example.com/search?q=alert` an example prefix for searching something like confluence

If this is overly complex let me know and I'll remove the hash.

Reply:
I think that's a reasonable decision.